src/components/item-image/index.js

```diff
@@ -282,6 +284,19 @@ function ItemImage({
         };
     }, [item, linkToItem, navigate]);
 
+    const imageText = useMemo(() => {
+        let element = <div style={imageTextStyle} onClick={imageTextClick}>{item.shortName}</div>;
+        if (!element) {
+            return element;
+        }
```

Comment:
What this is for? I think `element` will be always something valorized

Reply:
I think I added that when I was troubleshooting something. Removed now, thanks!